docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeSupervisedBinning.cs

```diff
@@ -0,0 +1,89 @@
+            var mlContext = new MLContext();
+            var samples = new List<DataPoint>()
+            {
+                new DataPoint(){ Features = new float[4] { 8, 1, 3, 0}, Bin="Bin1" },
+                new DataPoint(){ Features = new float[4] { 6, 2, 2, 1}, Bin="Bin2" },
+                new DataPoint(){ Features = new float[4] { 5, 3, 0, 2}, Bin="Bin2" },
+                new DataPoint(){ Features = new float[4] { 4,-8, 1, 3}, Bin="Bin3" },
+                new DataPoint(){ Features = new float[4] { 2,-5,-1, 4}, Bin="Bin3" }
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+            // Let's transform "Bin" column from string to key.
+            data = mlContext.Transforms.Conversion.MapValueToKey("Bin").Fit(data).Transform(data);
+            // NormalizeSupervisedBinning normalizes the data by constructing bins based on correlation with the label column and produce output based on 
+            // to which bin original value belong.
+            var normalize = mlContext.Transforms.NormalizeSupervisedBinning("Features", labelColumnName: "Bin", mininimumExamplesPerBin: 1, fixZero: false);
+
+            // NormalizeSupervisedBinning normalizes the data by constructing bins based on correlation with the label column and produce output based on 
+            // to which bin original value belong but make sure zero values would remain zero after normalization.
+            // Helps preserve sparsity.
+            var normalizeFixZero = mlContext.Transforms.NormalizeSupervisedBinning("Features", labelColumnName: "Bin", mininimumExamplesPerBin: 1, fixZero: true);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var normalizeTransform = normalize.Fit(data);
+            var transformedData = normalizeTransform.Transform(data);
+            var normalizeFixZeroTransform = normalizeFixZero.Fit(data);
+            var fixZeroData = normalizeFixZeroTransform.Transform(data);
+            var column = transformedData.GetColumn<float[]>("Features").ToArray();
+            foreach (var row in column)
+                Console.WriteLine(string.Join(", ", row.Select(x => x.ToString("f4"))));
+            // 1.0000, 0.5000, 1.0000, 0.0000
+            // 0.5000, 1.0000, 0.0000, 0.5000
+            // 0.5000, 1.0000, 0.0000, 0.5000
+            // 0.0000, 0.0000, 0.0000, 1.0000
+            // 0.0000, 0.0000, 0.0000, 1.0000
+
+            var columnFixZero = fixZeroData.GetColumn<float[]>("Features").ToArray();
+            foreach (var row in columnFixZero)
+                Console.WriteLine(string.Join(", ", row.Select(x => x.ToString("f4"))));
+            // 1.0000, 0.0000, 1.0000, 0.0000
+            // 0.5000, 0.5000, 0.0000, 0.5000
```

Comment:
nit: could you add
// Expected output:

Reply:
I suspect expected values are incorrect: input:  2, 2, 2, 0 minmaxnormalized returns 1,1,1,0 or am I missing something?